src/MusicalScore/Graphical/VexFlow/VexFlowVoiceEntry.ts

```diff
@@ -9,32 +10,32 @@ import { ColoringModes } from "./../DrawingParameters";
 import { GraphicalNote } from "../GraphicalNote";
 
 export class VexFlowVoiceEntry extends GraphicalVoiceEntry {
-    private mVexFlowStaveNote: Vex.Flow.StemmableNote;
+    private mVexFlowStaveNote!: StemmableNote;
 
-    constructor(parentVoiceEntry: VoiceEntry, parentStaffEntry: GraphicalStaffEntry) {
+    constructor(parentVoiceEntry?: VoiceEntry, parentStaffEntry?: GraphicalStaffEntry) {
         super(parentVoiceEntry, parentStaffEntry);
     }
 
     public applyBordersFromVexflow(): void {
-        const staveNote: any = (this.vfStaveNote as any);
-        if (!staveNote.getNoteHeadBeginX) {
+        const staveNote: StemmableNote = this.vfStaveNote;
+        if (!(staveNote instanceof Vex.Flow.StaveNote)) {
             return;
         }
-        const boundingBox: any = staveNote.getBoundingBox();
-        const modifierWidth: number = staveNote.getNoteHeadBeginX() - boundingBox.x;
+        const boundingBox: BoundingBox = staveNote.getBoundingBox();
```

Comment:
import as VFBoundingBox

Reply:
Yes. There might be more naming conflicts with Vexflow, so prefix the Vexflow versions with VF.

Reply:
It might actually be nice to have the VF prefix for (almost?) every Vexflow class, where we had Vex.Flow.X before.

Reply:
I have done it on all the classes with collision. This one was missing.